docs/local.sh

```diff
@@ -33,4 +33,4 @@ docker run --rm \
         -w="/app" \
         -p 1313:1313 \
         android/docs/local:latest \
-        sh -c "hugo server --cleanDestinationDir --i18n-warnings --minify --theme book --bind 0.0.0.0"
+        sh -c "hugo server --cleanDestinationDir --i18n-warnings --minify --bind 0.0.0.0 --baseURL http://localhost:1313/avito-android"
```

Comment:
`--theme book` - how it will work without specified theme?

Reply:
The theme is already defined in config
https://github.com/avito-tech/avito-android/blob/develop/docs/config.toml#L12